modules/codegen/src/main/scala/com/twilio/guardrail/generators/syntax/Java.scala

```diff
@@ -65,6 +66,10 @@ object Java {
     def toNodeList: NodeList[T] = new NodeList[T](l: _*)
   }
 
+  implicit class RichNodeList[T <: Node](val nl: NodeList[T]) extends AnyVal {
+    def toList(implicit cls: ClassTag[T]): List[T] = List(nl.iterator.asScala.toArray: _*)
```

Comment:
`toList` instead of `toArray`?

Reply:
Is that better?  My feeling is that since varargs is (under the hood) just an array, why convert to List just so the compiler/JVM can convert to Array?

Reply:
```scala
    def toList(implicit cls: ClassTag[T]): List[T] = List(nl.iterator.asScala.toArray: _*)
```
==
```scala
    def toList(implicit cls: ClassTag[T]): List[T] = nl.iterator.asScala.toList
```
no?

Reply:
wow.  did not see that in the autocomplete list.  i's dumb.